examples/websocket-client-mutual-ssl/websocket_client_mutual_ssl.md

```diff
@@ -4,6 +4,14 @@ Ballerina supports mutual SSL, which is a certificate-based authentication proce
 
 For more information on the underlying module, see the [`websocket` module](https://lib.ballerina.io/ballerina/websocket/latest/).
 
+>**Tip:** You may need to change the certificate file path, private key file path, and trusted certificate file path in the code below.
+
 ::: code websocket_client_mutual_ssl.bal :::
 
-::: out websocket_client_mutual_ssl.out :::
\ No newline at end of file
+Run the client program by executing the command below.
+
+>**Info:** As a prerequisite to running the client, start a sample service secured with mutual SSL.
+
+::: out websocket_client_mutual_ssl.out :::
+
+>**Info:** As a prerequisite to running the client, start a [sample service secured with Mutual SSL](/learn/by-example/websocket-service-mutual-ssl/).
```

Comment:
The prerequisite is duplicated here. please check

Reply:
Fixed.